nuxeo-distribution/nuxeo-server-tomcat/src/main/resources/tomcat/bin/nuxeo.conf

```diff
@@ -108,7 +108,7 @@ elasticsearch.httpEnabled=true
 
 ## Comma separated list of CorePageProvider to supersede by Elasticsearch.
 ##  default is default_search,DEFAULT_DOCUMENT_SUGGESTION.
-#elasticsearch.override.pageproviders=default_search,document_content,section_content,document_content,tree_children,default_document_suggestion,simple_search,advanced_search,nxql_search,DEFAULT_DOCUMENT_SUGGESTION
+#elasticsearch.override.pageproviders=default_search,section_content,document_content,tree_children,default_document_suggestion,simple_search,advanced_search,nxql_search,DEFAULT_DOCUMENT_SUGGESTION,REST_API_SEARCH_ADAPTER,all_collections
```

Comment:
Here I would not try to maintain a list of possible options. And the mention of the default is actually wrong. So maybe just write:

    ## Comma separated list of CorePageProvider to supersede by Elasticsearch.
    ## The default is defined by nuxeo.defaults in template common-base
    #elasticsearch.override.pageproviders=...

Reply:
Or even simpler remove these lines.